src/main/java/core/basesyntax/dao/StorageDao.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface StorageDao {
+    Map<String, Integer> save(String fruit, int quantity);
```

Comment:
The `save` method returns a `Map<String, Integer>`, which might not be necessary if the purpose is only to update the storage. Consider returning a more appropriate type or void if the map is not needed after saving.